.eslintrc

```diff
@@ -14,7 +14,8 @@
     "consistent-return": 0,
     "comma-dangle": 0,
     "spaced-comment": 0,
-    "global-require": 0
+    "global-require": 0,
+    "no-param-reassign": ["error", {"props": false}]
```

Comment:
Fascinating. My gut was to disagree but you're right. I guess ideally we'd look at refactoring our code to be more in line with what AirBnb had in mind, because I _do_ like following the "no param reassign" rule. I.e. make our functions pure functions, that never modify the sent state.

But just removing the rule is fine too.